Sources/DVR/Session.swift

```diff
@@ -163,6 +182,25 @@ open class Session: URLSession {
         }
     }
 
+    func finishTaskAndDoNotPersist(_ task: URLSessionTask, responseData: Data?, playback: Bool) {
+        needsPersistence = (needsPersistence || !playback)
```

Comment:
Same question here. It seems to me the question of "to persist or not to persist" should have been resolved when deciding which method to call.